src/types/History.ts

```diff
@@ -0,0 +1,12 @@
+type History = {
+  id: number,
+  steps: 
+  {
+    step: number,
+    status: string,
+    description: string,
+    date: string
```

Comment:
I think breaking this part into two types would make it clearer. No need to create another file, just putting `Step` type above `History` works fine :)